src/api/app/components/bs_request_comment_component.html.haml

```diff
@@ -24,7 +24,7 @@
                                     action: comment_path(comment) },
                             class: 'dropdown-item delete_link', title: 'Delete comment') do
                 Delete
-      .px-3.py-2.border-bottom
+      .px-3.py-2.border-bottom.d-flex.flex-column.flex-lg-row.justify-content-between.align-items-start
```

Comment:
This makes the badges appear below the username and the timestamp for smaller screens